sbt.bat

```diff
@@ -0,0 +1,33 @@
+@REM https://github.com/xerial/sbt-pack/blob/master/src/main/templates/launch-bat.mustache
+@REM would be worth getting more inspiration from
+
+@echo off
+
+SET ERROR_CODE=0
+
+SET LAUNCHER_PATH=%localappdata%/coursier
```

Comment:
This reminds me, I've had to change this to:
```
SET LAUNCHER_PATH=%localappdata%/coursier.jar
```
when running this in windows. This is because when courier runs it tries to create a directory in this same path called `Coursier` which fails because windows thinks that there's already a directory there with the same name but its really the jar that was downloaded.

Reply:
Sure, I can rename it. I've never tried it on windows and copied most of these files from [slamdata/backend.g8](https://github.com/slamdata/backend.g8).